Jenkinsfile

```diff
@@ -17,10 +17,8 @@ node('docker'){
 
     stage ('publish') {
       withCredentials([usernamePassword(credentialsId: 'GitHub-Jenkins', usernameVariable: 'GIT_USERNAME', passwordVariable: 'GIT_PASSWORD')]) {
-        withCredentials([usernamePassword(credentialsId: 'artifactory-modelon', usernameVariable: 'PYPI_USERNAME', passwordVariable: 'PYPI_PASSWORD')]) {
+        withCredentials([usernamePassword(credentialsId: 'pypi-modelon-api-key')]) {
           env.GIT_CREDENTIALS = "${GIT_USERNAME}:${GIT_PASSWORD}"
-          env.PYPI_USERNAME = "${PYPI_USERNAME}"
-          env.PYPI_PASSWORD = "${PYPI_PASSWORD}"
```

Comment:
We will still have these environment variables, they will contain the username \_\_token\_\_ and the API key.